eloquentjs/11_1_quiet_times/quiet_times.js

```diff
@@ -0,0 +1,47 @@
+const { generateLogs } = require('./src/generateLogs');
+const { textFile } = require('./src/textFile');
+
+/**
+ * Returns promise of hourly security camera activity recorded for given day of week (0-6)
+ * @param {*} day 
+ * @returns 
+ */
+async function activityTable(day) {
+    const camera_logs = await textFile('camera_logs.txt');
+    const filePaths = camera_logs.split('\n').filter(j => j);
+
+    const timeStamps = [];
+    for (let i = 0; i < filePaths.length; i++) {
+        const logContents = await textFile(`weekly_logs/${filePaths[i]}`);
+
+        logContents.split('\n').filter(j => j).forEach(element => {
```

Comment:
This would be clearer if you formatted it with one chained method call per line like this
```js
logContents
  .split('\n')
  .filter...
```

Also, `filter(j => j)` isn't the best way to express the idea I think you're expressing since it relies on JS's weird truthy value semantics. `filter(j => j.length > 0)` makes the intent I think you have much more explicit.